apps/vaporgui/VDoubleSpinBox.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
```

Comment:
What's the purpose of this pragma? Is there a guideline of when and where to use it?

Reply:
It prevents a .h file from being included multiple times within a single compilation unit.

Reply:
Shall we use include guards instead, per our conventions?

Reply:
I think we should change our conventions to use "pragma once".  It's superior to include guards because include guards rely on the #define being a unique name among all of the include dependencies.